setup.py

```diff
@@ -8,6 +8,17 @@
   pass
 
 try:
+  from subprocess import STDOUT, check_call
+  import os
+  check_call(['apt-get', 'install', '-y', 'libhdf5-serial-dev'])
```

Comment:
This is Debian-specific, so if you may want to change it to include os x instead.